opencti-worker/src/worker.py

```diff
@@ -643,7 +645,8 @@ def __post_init__(self) -> None:
         # Initialize variables
         self.connectors: List[Any] = []
         self.queues: List[Any] = []
-        self.execution_pool = ThreadPoolExecutor(max_workers=self.opencti_pool_size)
+        global execution_pool
+        execution_pool = ThreadPoolExecutor(max_workers=self.opencti_pool_size)
         self.listen_api_execution_pool = ThreadPoolExecutor(
```

Comment:
self.listen_api_execution_pool is no more really used since self.execution_pool is no more used in ApiConsumer.
The global variable may add confusion, it would be clearer to remove self.list_api_execution_pool and provide self.execution_pool instead to ApiConsumer constructors

Reply:
Indeed, that was a pretty bad implementation here. Fixed, and we still use two separate execution queues to avoid head-of-line blocking if a callback URL is slow.